sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -1007,10 +997,147 @@ class RocksDB(
     }
   }
 
+  override protected def logName: String = s"${super.logName} $loggingId"
```

Comment:
What is the implication of this change ?

Reply:
This is an existing code. See line 1038. Just the way git is showing it given I added some new code around it.